package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "fec",
+  "version": "1.0.0",
+  "description": "FEC project",
+  "main": "webpack.config.js",
+  "dependencies": {
+    "express": "^4.17.1",
+    "mysql": "^2.17.1",
+    "react-dom": "^16.8.6",
+    "react": "^16.8.6",
+    "webpack-cli": "^3.3.5"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.5.0",
+    "@babel/preset-env": "^7.5.2",
+    "@babel/preset-react": "^7.0.0",
```

Comment:
You don't seem to be using Babel or webpack as devDependencies. I recommend putting them under `dependencies` instead.